modules/declarative/pages/policy.adoc

```diff
@@ -0,0 +1,293 @@
+
+NOTE: Each setting for the {policy-system} can be applied to one or more namespaces.
+
+For example, the following policy CR disallows outgoing links for all namespaces, but all other {skupper-name} capabilities are allowed:
+
+[source,yaml]
+----
+apiVersion: skupper.io/v1alpha1
+kind: SkupperClusterPolicy
+metadata:
+  name: cluster-policy-sample-01
+spec:
+  namespaces:
+    - "*"
+  allowIncomingLinks: true
+  allowGateway: true
+  allowedExposedResources:
+    - "*"
+  allowedOutgoingLinksHostnames: []
+  allowedServices:
+    - "*"
+----
+
+`namespaces`:: One or more patterns to specify the namespaces that this policy applies to.
+
+`allowIncomingLinks`:: Specify `true` or `false` to determine whether other sites can create links to the namespaces specified.
+
+`allowedOutgoingLinksHostnames`:: Specify one or more patterns to determine which hosts you can create links to from the namespaces specified.
+
+`allowedServices`:: Specify one or more patterns to determine the permitted names of services allowed on the {service-network} from the namespaces specified.
+
+`allowedExposedResources`:: Specify one or more patterns to determine the permitted names of service bindings allowed on the {service-network} from the namespaces specified.
+
+[TIP]
+====
+Use regular expressions to create matches, for example:
+
+* `.*/.com` matches any string ending in `.com`.
+* `^abc$` matches the string `abc`.
+
+====
+
+If you create another CR that allows outgoing links for a specific namespace, a user can create a link from that namespace to join a {service-network}, that is, the logic for multiple policy CRs is `OR`.
+{skupper-name} allows operations if one or more of the policy CRs permits the operation.
+
+// Type: procedure
+[id="installing-crd"] 
+== Installing the {policy-system} CRD
+
+Install the {policy-system} CRD enables a cluster administrator enforce policies for {service-network}s.
+
+NOTE: If there are existing sites on the cluster, see xref:installing-crd-existing-sites[]
+
+.Prerequisites
+
+* Access to a cluster using a `cluster-admin` account
+* The operator is installed in {OperatorLink}
+.Procedure
+
+. Log into the cluster using a `cluster-admin` account.
+
+. Download the CRD.
+
+. Apply the CRD:
++
+[source,yaml]
+----
+$ kubectl apply -f skupper_cluster_policy_crd.yaml
+
+customresourcedefinition.apiextensions.k8s.io/skupperclusterpolicies.skupper.io created
+clusterrole.rbac.authorization.k8s.io/skupper-service-controller created
+----
+
+
+. To verify that the {policy-system} is active, use the `skupper status` command and check that the output includes the following line:
++
+[source,bash]
+----
+Skupper policy is enabled
+----
+
+
+// Type: procedure
+[id="installing-crd-existing-sites"] 
+== Installing the {policy-system} CRD on a cluster with existing sites
+
+If the cluster already hosts {skupper-name} sites, note the following before installing the CRD:
```

Comment:
... and the existing sites are already running Skupper > 0.8 (which introduces the policy system).
If sites are still running 0.8, then you can create CRs before updating them.

Reply:
I created a section for this case